src/ripple/protocol/impl/STAmount.cpp

```diff
@@ -1261,6 +1261,8 @@ divRound (STAmount const& num, STAmount const& den,
 std::uint64_t
 mulDiv(std::uint64_t value, std::uint64_t mul, std::uint64_t div)
 {
+    if ((value == 0 || mul == 0) && div != 0)
+        return 0;
```

Comment:
This may change transaction processing. Prove that it doesn't, or remove this change.